mautic.bc.yaml

```diff
@@ -10,16 +10,16 @@ objects:
       annotations:
         openshift.io/generated-by: OpenShiftNewBuild
       labels:
-        build: ${NAME}
-      name: ${NAME}
+        build: ${NAME}-${PR}-${SUFFIX}
+      name: ${NAME}-${PR}-${SUFFIX}
```

Comment:
I'd leave the image stream as is with no suffix. It can store all the image tags